dao/src/main/java/se/fortnox/reactivewizard/db/transactions/TransactionStatement.java

```diff
@@ -1,35 +1,19 @@
 package se.fortnox.reactivewizard.db.transactions;
 
-import se.fortnox.reactivewizard.db.ConnectionProvider;
 import se.fortnox.reactivewizard.db.statement.Statement;
 
 import java.sql.Connection;
 import java.sql.SQLException;
-import java.util.concurrent.atomic.AtomicReference;
 
 public class TransactionStatement implements Batchable {
```

Comment:
This is really optional, but it is possible to drop this class if you let `Statement` implement `Batchable`.